net/unbound/Makefile

```diff
@@ -122,14 +122,14 @@ endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/unbound.h $(1)/usr/include/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/include/unbound.h $(1)/usr/include/
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libunbound.{so*,a,la} $(1)/usr/lib/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/libunbound.{so*,a,la} $(1)/usr/lib/
```

Comment:
> .so files are expected to have executable bit set and INSTALL_DATA will remove them I guess. So this part might be wrong.

On a Debian system they don't have the executable bit and on OpenWrt it's mixed (`/lib/libfstools.so` is `-rw-r--r--` and `/lib/libuci.so` is `-rwxr-xr-x`). Technically speaking there's no reason for the executable bit being set on shared libraries.

> the archive flag set

I'm only familiar with the archive bit on Windows filesystems. What are you referring to in regards to POSIX filesystems?